developing.adoc

```diff
@@ -0,0 +1,45 @@
+
+
+= Developing CodeReady Containers
+
+[[developing-overview]]
+== Overview
+
+The following sections describe how to build and test {project}.
+
+[[develop-prerequisites]]
+== Prerequisites
+
+- Git
+- Make
+- A recent Go distribution (>=1.11)
+
+[NOTE]
+====
+You should be able to develop the project on Linux, Windows or macOS.
+====
+
+[[set-up-dev-env]]
+== Setting Up the Development Environment
+
+[[cloning-repository]]
+=== Cloning the Repository
+
+Get the sources from GitHub:
+
+----
+$ git clone https://github.com/code-ready/crc.git
+----
+
+Note: Do not keep the source in your $GOPATH as it will cause `make` to fail. Because CRC uses link:https://github.com/golang/go/wiki/Modules[go modules] and it does not expect the source to be in the $GOPATH.
```

Comment:
I'd use a block admonition here. Also, do we need to explain the reason behind `make` failing, or would it suffice to simply explain that the source should be outside of your `$GOPATH`?

Reply:
>  would it suffice to simply explain that the source should be outside of your `$GOPATH`?

This enough as `go modules` expect it that way.